common/src/lib/java/dev/engine_room/flywheel/lib/model/baked/MultiBlockModelBuilder.java

```diff
@@ -24,15 +23,11 @@ public abstract class MultiBlockModelBuilder {
 	@Nullable
 	BiFunction<RenderType, Boolean, Material> materialFunc;
 
-	MultiBlockModelBuilder(BlockAndTintGetter level, Iterable<BlockPos> positions) {
+	public MultiBlockModelBuilder(BlockAndTintGetter level, Iterable<BlockPos> positions) {
 		this.level = level;
 		this.positions = positions;
 	}
 
-	public static MultiBlockModelBuilder create(BlockAndTintGetter level, Iterable<BlockPos> positions) {
```

Comment:
Do we trust that we won't need these factory methods in the future? It's easy to change what the contents of a function point to while changing constructors is much more delicate.